DigitalLearningSolutions.Data/Enums/TrackerEndpointAction.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace DigitalLearningSolutions.Data.Enums
+{
+    using System;
+
+    public class TrackerEndpointAction : Enumeration

```

Comment:
I've thought about this a little more, and I think it makes more sense for this to be an enum, so it can be used in the switch statement in `TrackerService.ProcessQuery`.